data-raw/simulate_longitudinal_data.R

```diff
@@ -185,9 +175,9 @@ for (i in 1:12) {
   dfParticipants <-
     AddParticipants(
       dfParticipants,
-      studyid = "ABC-123",
+      studyid = "AA-AA-000-0000",
```

Comment:
Maybe I didn't have a fresh session or something with the set.seed() situation but my results & etc. don't quite perfectly match what we originally had so this had small implications for the testthat stuff

Reply:
@lauramaxwell in particular I'd probably need a solid audit/double check what was done here in `data-raw/simulate_longitudinal_data.R`, can set the for loop to (i in 1:2) if you want to speed up/check the structure of everything inside is fed and done correctly

Reply:
taking a look now!

Reply:
in looking into this, i think i originally overcomplicated the country-fication of the simulated data. i'm going to dig into this this afternoon and see if can simplify the process. The updating you did for v2.1 updates looks great, so that will definitely stay.

Reply:
okay, after making some changes, I came back around to this approach in the end, so we're going to keep it. re: testing differences, it makes sense that there are differences because even with the same seed, you had to add some lines of code, which impacted the R session so that sampling was not identical.